azure-pipelines.yml

```diff
@@ -50,29 +50,43 @@ jobs:
                        ripgrep pyflakes beautifulsoup4 chardet pycrypto glob2 psutil pytz tqdm  \
                        conda-package-handling py-lief python-libarchive-c perl conda-forge::shellcheck
       conda install -q pytest pip pytest-cov pytest-forked pytest-xdist nomkl numpy mock pytest-mock
+      conda install -q pytest-replay -c conda-forge
```

Comment:
@msarahan , can we have ``pytest-replay`` on the main channel?

Reply:
I'll add it to the queue